src/commands/networks.ts

```diff
@@ -30,26 +32,38 @@ export const supportedNetwork: {
   [NetworkCmdName.Local]: {
     explorer: "https://localhost/explorer",
     provider: jsonRpcProvider("http://127.0.0.1:8545"),
+    networkId: 1337,
+    networkName: "local",
   },
   [NetworkCmdName.Mainnet]: {
     explorer: "https://etherscan.io",
     provider: defaultInfuraProvider("homestead"),
+    networkId: 1,
+    networkName: "homestead",
   },
   [NetworkCmdName.Goerli]: {
     explorer: "https://goerli.etherscan.io",
     provider: defaultInfuraProvider("goerli"),
+    networkId: 5,
+    networkName: "goerli",
   },
   [NetworkCmdName.Sepolia]: {
     explorer: "https://sepolia.etherscan.io",
     provider: jsonRpcProvider("https://rpc.sepolia.org"),
+    networkId: 11155111,
+    networkName: "sepolia",
   },
   [NetworkCmdName.Polygon]: {
     explorer: "https://polygonscan.com",
     provider: defaultInfuraProvider("matic"),
+    networkId: 137,
+    networkName: "matic",
   },
   [NetworkCmdName.Mumbai]: {
     explorer: "https://mumbai.polygonscan.com",
     provider: defaultInfuraProvider("maticmum"),
+    networkId: 80001,
+    networkName: "maticmum",
```

Comment:
can explain to me the diff between `networkName` and `networkCmdName`?
i see that the only diff is mumbai's name and mainnet's name is diff

Reply:
hmm seems abit duplicative but its okay, thx for explaining